requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy
```

Comment:
do we need this?

It's my undestanding that for a single pacakge, you put the requrements in the setup.py

requirements.txt is for applications, or systems, or ... stuff that is not a single package.

the idea is that pip install this_package will be able to resolve the dependencies.